Fixed recursive send bug. Now files can be sent even if they are smaller then the FILE_CHUNK_SIZE

hitecSmartHome opened the pull request:
from `err = send();` to `err = _response->send();`

mathieucarbou commented:
@hitecSmartHome : if @hoeken agrees with, I would also be in favour for drastically reducing the FILE_CHUNK_SIZE (to 1024 maybe ?) - but in another PR.

8K is very high especially if serving requests concurrently. I think it is best to keep a low number by default that will work most of the cases and leave the opportunity to users to increase it if they have the ability to do so.

mathieucarbou commented:
Also, i've changed the target branch to v2 ;-)

hitecSmartHome commented:
Yeah, I have also changed it in my version to

```cpp
#ifndef FILE_CHUNK_SIZE
  #define FILE_CHUNK_SIZE 1 * 1024
#endif
```

> Also, i've changed the target branch to v2 ;-)

Sorry, I didn't see that I was creating the pull against the V1
